### ISSUE TITLE
Earl Grey unable to detect the Accessibility Issues

### ISSUE THREAD

**Issue by Prabhu1229** [google/GTXiLib#18]
I used EarlGrey 2.0 and I could not able to see any failure accessibility failure although I set by button frame to 5*5

can any one help me here @niksawtschuk @j-sid @selin194
